com.sublimetext.three.json

```diff
@@ -1,11 +1,11 @@
 {
     "app-id": "com.sublimetext.three",
     "runtime": "org.freedesktop.Sdk",
-    "runtime-version": "1.6",
+    "runtime-version": "18.08",
     "sdk": "org.freedesktop.Sdk",
     "base": "io.atom.electron.BaseApp",
-    "base-version": "stable",
-    "branch": "stable",
+    "base-version": "18.08",
+    "branch": "18.08",
```

Comment:
This actually changes the branch name of the app that will be exported to Flathub, that's probably not what we want, this should remain `stable`.

Reply:
Yes its forced to be `stable`, anything else is invalid.